build/source/driver/multi_driver.f90

```diff
@@ -640,41 +813,31 @@ subroutine handle_err(err,message)
  endif
  ! dump variables
  print*, 'error, variable dump:'
- if(allocated(dt_init)) print*, 'dt = ', dt_init
- print*, 'istep = ', istep
- if(associated(type_data))then
-  print*, 'HRU index = ', type_data%var(iLookTYPE%hruIndex)
- endif
- if(associated(forc_data))then
-  print*, 'pptrate            = ', forc_data%var(iLookFORCE%pptrate)
-  print*, 'airtemp            = ', forc_data%var(iLookFORCE%airtemp)
- endif
- if(associated(mpar_data))then
-  print*, 'theta_res         = ', mpar_data%var(iLookPARAM%theta_res)            ! soil residual volumetric water content (-)
-  print*, 'theta_sat         = ', mpar_data%var(iLookPARAM%theta_sat)            ! soil porosity (-)
-  print*, 'plantWiltPsi      = ', mpar_data%var(iLookPARAM%plantWiltPsi)         ! matric head at wilting point (m)
-  print*, 'soilStressParam   = ', mpar_data%var(iLookPARAM%soilStressParam)      ! parameter in the exponential soil stress function (-)
-  print*, 'critSoilWilting   = ', mpar_data%var(iLookPARAM%critSoilWilting)      ! critical vol. liq. water content when plants are wilting (-)
-  print*, 'critSoilTranspire = ', mpar_data%var(iLookPARAM%critSoilTranspire)    ! critical vol. liq. water content when transpiration is limited (-)
- endif
- if(associated(mvar_data))then
-  if(associated(mvar_data%var(iLookMVAR%scalarSWE)%dat))then
-   print*, 'scalarSWE = ', mvar_data%var(iLookMVAR%scalarSWE)%dat(1)
-   print*, 'scalarSnowDepth = ', mvar_data%var(iLookMVAR%scalarSnowDepth)%dat(1)
-   print*, 'scalarCanopyTemp = ', mvar_data%var(iLookMVAR%scalarCanopyTemp)%dat(1)
-   print*, 'scalarRainPlusMelt = ', mvar_data%var(iLookMVAR%scalarRainPlusMelt)%dat(1)
-   write(*,'(a,100(i4,1x))'   ) 'layerType          = ', indx_data%var(iLookINDEX%layerType)%dat
-   write(*,'(a,100(f11.5,1x))') 'mLayerDepth        = ', mvar_data%var(iLookMVAR%mLayerDepth)%dat
-   write(*,'(a,100(f11.5,1x))') 'mLayerTemp         = ', mvar_data%var(iLookMVAR%mLayerTemp)%dat
-   write(*,'(a,100(f11.5,1x))') 'mLayerVolFracIce   = ', mvar_data%var(iLookMVAR%mLayerVolFracIce)%dat
-   write(*,'(a,100(f11.5,1x))') 'mLayerVolFracLiq   = ', mvar_data%var(iLookMVAR%mLayerVolFracLiq)%dat
-   print*, 'mLayerMatricHead   = ', mvar_data%var(iLookMVAR%mLayerMatricHead)%dat
-   print*, 'column inflow = ', mvar_data%var(iLookMVAR%mLayerColumnInflow)%dat
-
-  endif
+ print*, 'istep              = ', istep
```

Comment:
Is there a way in fortran to print this to `stderr` rather than `stdout`? Will make it easier when redirecting output streams to different files.